docs/integrations/ integration-statuses.md

```diff
@@ -8,6 +8,6 @@ Here is a list of possible statuses for Macrometa connections. You can [view the
 - **Succeeded**: The connection completes successfully, it transitions to the Succeeded state. This status means that all the steps and tasks within the connection have been executed without any errors or failures.
 - **Running**: The connection is currently being executed and progressing towards completion.
 - **Failed**: If any step or task within the connection encounters an error or failure, then the connection transitions to the Failed state. This status indicates that the connection execution was unsuccessful and requires attention to diagnose and resolve the underlying issues.
-- **Error**: The Error status is similar to the Failed state, but it represents a broader range of errors that may occur outside of the individual steps or tasks. It could indicate issues with the Central Cloud, connectivity problems, or any other critical errors that prevent proper connection execution.
+- **Error**: The Error status is similar to the Failed state, but it represents a broader range of errors that may occur outside of the individual steps or tasks. It could indicate issues with the central cloud, connectivity problems, or any other critical errors that prevent proper connection execution.
```

Comment:
@grainier-macrometa @diana-macrometa is central cloud term supposed to be used?

Reply:
@dustinlarimer and @elof, seems like a question for you guys. We don't use it elsewhere, but I thought it might make sense in the integration sense. Central cloud being the external resource...

Reply:
@diana-macrometa @dustinlarimer I think calling it something like the origin data service. It could be a private data center.

Reply:
Updated, please review.